src/components/gravid/GravidSide.scss

```diff
@@ -4,6 +4,18 @@
     margin-top: 0;
 }
 
+.gravidside-alert-gravid {
```

Comment:
Skal vi kjøre blanding av sass og scss? Ikke kjeempeviktig sak...

Reply:
For min skyld helst ikke, men denne var allerede aktiv, så i stedet for å refaktor her valgte jeg å bruke den